src/oss-find-squats-lib/Mutators/IMutator.cs

```diff
@@ -35,6 +36,12 @@ public IEnumerable<Mutation> Generate(PackageURL arg)
             bool hasNamespace = arg.HasNamespace();
             foreach (Mutation mutation in Generate(hasNamespace ? arg.Namespace : arg.Name))
             {
+                if (mutation.Mutated.Length == 1 &&
+                    SeparatorRemovedMutator.DefaultSeparators.Contains(Convert.ToChar(mutation.Mutated)))
```

Comment:
I wonder if checking if the single character is alphanumeric might be better here rather than if it's just one of the default separators. Do we know of any manager where a single non alphanumeric would be valid as an identifier?

https://learn.microsoft.com/en-us/dotnet/api/system.char.isletterordigit?view=net-8.0